pandas/core/tools/timedeltas.py

```diff
@@ -68,7 +68,7 @@ def to_timedelta(arg, unit=None, errors="raise"):
 
     Returns
     -------
-    timedelta64 or numpy.array of timedelta64
+    Timedelta or TimedeltaIndex of timedelta64
```

Comment:
This can also return a Series of `timedelta64[ns]` if a `Series` is the input.

Would be great if you could organize this `Returns` sections like the `Returns` section of `to_datetime`